From d55febfa25c569d3b9286addaacdf54ce16f2684 Mon Sep 17 00:00:00 2001
From: Wulf C. Krueger <philantrop@exherbo.org>
Date: Sun, 21 Nov 2010 12:48:37 +0100
Subject: [PATCH 37/40] bootchart2: systemd parts of the install target need to respect DESTDIR

Hello!

When trying to install the systemd services, the Makefile currently doesn't
use DESTDIR. Attached you'll find a patch to fix that.

Best regards, Wulf

From 602008f3a47c6a03dd948bcdc7dca4caa14a7e24 Mon Sep 17 00:00:00 2001
From: Wulf C. Krueger <philantrop@exherbo.org>
Date: Sun, 21 Nov 2010 12:12:38 +0100
Subject: [PATCH] Respect DESTDIR for the systemd parts of the install target.
---
 Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 00b860b..a720976 100644
--- a/Makefile
+++ b/Makefile
@@ -45,11 +45,11 @@ install-collector: all install-chroot
 	install -m 755 -D bootchart-collector $(DESTDIR)/lib/bootchart/bootchart-collector
 
 install-service:
-	mkdir -p $(SYSTEMD_UNIT_DIR)
+	mkdir -p $(DESTDIR)/$(SYSTEMD_UNIT_DIR)
 	install -m 0644 bootchart.service \
 	       bootchart-done.service \
 	       bootchart-done.timer \
-	       $(SYSTEMD_UNIT_DIR)      
+	       $(DESTDIR)/$(SYSTEMD_UNIT_DIR)      
 
 install: all py-install-compile install-collector install-service
 
-- 
1.7.3.1

